application/src/main/resources/application-authorization.properties

```diff
@@ -0,0 +1,4 @@
+authorization.service.host=http://localhost:8081
```

Comment:
Maybe we should remove this config file as well, and only keep this kind of configuration in services or other plugins that implement different types of authorizations

Reply:
okay, i remove it